app/routes.js

```diff
@@ -6,6 +6,6 @@ const health = require("./routes/health");
 module.exports = {
   index,
   success,
-  qrcode,
+  qrcode, // PROTOTYPE LINE
```

Comment:
What about the import in line 3?

Reply:
I thought, that imports are almost automatically done via IDE, but to be complete, we can add it, you're right.